components/open-map-button.tsx

```diff
@@ -0,0 +1,25 @@
+import { SecondaryButton } from "./ui/button";
+
+import { MapIcon } from "@heroicons/react/outline";
+
+export function OpenMapButton({ address }: { address: string }) {
+  function btnClick(e: React.FormEvent<HTMLButtonElement>) {
+    e.currentTarget.blur();
+    window.open(`https://www.google.com/maps/search/?api=1&query=${address}`);
+  }
+
+  return (
+    <div className="flex items-start">
+      <SecondaryButton
+        aria-label="Buka Map"
+        className="relative z-10"
+        icon={MapIcon}
+        onClick={btnClick}
+        size="xs"
+        type="button"
+      >
+        Buka Map
```

Comment:
```suggestion
        Buka Peta
```

To make it consistent for the users, let's use Bahasa Indonesia words 😁